lib/langs/python.rb

```diff
@@ -56,7 +58,32 @@ def parse_requirements(content)
     end
 
     def parse_pipfile(content)
-      []
+      begin
+        config = TomlRB.parse(content)
+      rescue TomlRB::ParseError => e
+        log_error("Encountered error when parsing Pipfile: #{e.class} #{e.message}")
+        return []
+      end    
+      config.select! {|k,v| ['packages', 'dev-packages'].include?(k)}
+      results = []
+
+      config.each do |type, packages|
+        packages.each do |name, info|
+          requirement = info.is_a?(String) ? info : info.fetch('version', nil)
```

Comment:
An alternative here, that doesn't check for String, is this:
```ruby
requirement = info.fetch('version', nil) rescue info
```
If String coercion is important, you could change `rescue info` to `rescue info.to_s`.